tests/core/notification/test_events_published.py

```diff
@@ -156,6 +157,14 @@ def test_events_published_for_writing_dn():
 
 
 def test_events_published_for_scenario_submission():
+    _events_published_for_scenario_submission()
+
+def test_events_published_for_scenario_submission_standalone():
+    _events_published_for_scenario_submission(standalone=True)
+
+def _events_published_for_scenario_submission(standalone=False):
```

Comment:
Using the `@pytest.mark.parametrize(mode, ["development", "standalone"])` here would be cleaner.

However, this seems ok as well.